avocado/utils/vmimage.py

```diff
@@ -585,6 +588,7 @@ def download(self):
             cache_dirs = [self.cache_dir]
         else:
             cache_dirs = self.cache_dir
+        LOG.debug(f"Attempting to download image from URL: {self.url}")
```

Comment:
Using fstrings for logging is a big debate, but we've decided to folow the default pylint setting (see cf7574508b7de1bd7fbff7ec29e89fa054edec55).

And, these log lines with fstrings are the only ones under `avocado/utils`:

```
$ git grep LOG avocado/utils/ | grep f\" 
avocado/utils/vmimage.py:        LOG.debug(f"Attempting to download image from URL: {self.url}")
avocado/utils/vmimage.py:    LOG.debug(f"Provider for {name} not available")
```

So, for the sake of aautils, it's better to use old style logging here.